docs/tutorials/pydy_for_autolev_users.rst

```diff
@@ -0,0 +1,672 @@
+| y=2)                  | 2)])                  |                       |
++-----------------------+-----------------------+-----------------------+
+||                      ||                      ||                      |
+| P = Polynomial([a, b, | p = Poly(a*x**2 + b*x | For more information  |
+| c], x)                | + c)                  | refer to              |
+|                       |                       | `modules/polys. <htt  |
+|                       |                       | p://docs.sympy.org/la |
+|                       |                       | test/modules/polys/re |
+|                       |                       | ference.html>`__      |
++-----------------------+-----------------------+-----------------------+
+| Roots(Polynomial([a,  | solve(Poly(a*x**2 +   | For more information  |
+| b, c], x), x, 2)      | b*x +c), x)           | refer to `solvers. <ht| 
+|                       |                       | tp://docs.sympy.org/la|
+|                       |                       | test/modules/solvers/ |
+|                       |                       | solvers.html>`__      |
+|                       |                       |                       |
+|                       |                       | For numerical         |
+|                       |                       | computation related   |
+|                       |                       | to polynomials and    |
+|                       |                       | roots refer to        |
+|                       |                       | `mpmath/calculus. <htt|
+|                       |                       | p://docs.s            | 
+|                       |                       | ympy.org/0.7.6/module |
+|                       |                       | s/mpmath/calculus/pol |
+|                       |                       | ynomials.html>`__     |
++-----------------------+-----------------------+-----------------------+
+| Solve(A, x1, x2)      | linsolve(A, (x1, x2)) | For more information  |
+|                       |                       | refer to              |
+| where A is an         | where A is an         | `solvers/solveset. <ht| 
+| augmented matrix that | augmented matrix      | tp://docs.sympy.org/l |
+| represents the linear |                       | atest/modules/solvers |
+| equations and x1, x2  |                       | /solveset.html>`__    |
+| are the variables to  |                       |                       |
+| solve for.            |                       |                       |
+|                       |                       |                       |
++-----------------------+-----------------------+-----------------------+
+| RowMatrix = [1, 2, 3, | RowMatrix =           | For more information  |
+| 4]                    | Matrix([[1],[2],      | refer to `matrices. <h|
+|                       | [3],[4]])             | ttp://docs.sympy.org/ |
```

Comment:
I think it'd be nicer to follow pep8 naming conventions in the PyDy column as much as possible, i.e. `row_matrix = `